tests/old/users/test_umlaut_user.py

```diff
@@ -1,40 +0,0 @@
-from django.test.client import Client
-from django.test.utils import override_settings
-
-from openslides.users.models import Group, User
-from openslides.utils.test import TestCase
-
-
-@override_settings(PASSWORD_HASHERS=('django.contrib.auth.hashers.PBKDF2PasswordHasher',))
-class TestUmlautUser(TestCase):
-    """
-    Tests persons with umlauts in there name.
-    """
-
-    def setUp(self):
-        self.user = User.objects.create_user('äöü', 'äöü')
-        self.client = Client()
-        self.client.login(username='äöü', password='äöü')
-
-    def test_login(self):
```

Comment:
The first and the second test have no value. They only test the django login view.

The third test has to be removed.

We have to rewrite the login page to angular for the 2.0 in any case. So there is no harm to remove the hole file now.